src/pages/Main/MovieList/MovieList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import Movie from '../Movie/Movie';
+import './MovieList.scss';
+
+class MovieList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movieMoveCount: 0,
+    };
+  }
+
+  plusMoveCount = () => {
+    const { movieMoveCount } = this.state;
+    const { length } = this.props.data;
+    const maxCount = Math.ceil(length / 5 - 1);
+    if (length === 0) return;
+    if (movieMoveCount === maxCount) return;
+    this.setState({
+      movieMoveCount: movieMoveCount + 1,
+    });
+  };
+
+  minusMoveCount = () => {
+    const { movieMoveCount } = this.state;
+    if (movieMoveCount === 0) return;
+    this.setState({
+      movieMoveCount: movieMoveCount - 1,
+    });
+  };
+  render() {
+    const { movieMoveCount } = this.state;
+    const marginLeftValue = this.state.movieMoveCount * -1350;
+    const marginLeft = marginLeftValue.toString();
+    const { collectionMovies, data } = this.props;
+    const { minusMoveCount, plusMoveCount } = this;
+    const { length } = this.props.data;
+    const maxCount = Math.ceil(length / 5 - 1);
```

Comment:
```suggestion
    const { movieMoveCount } = this.state;
    const { collectionMovies, data } = this.props;
    const { minusMoveCount, plusMoveCount } = this;
    const marginLeft = this.state.movieMoveCount * -1350.toString();
    const maxCount = Math.ceil(data / 5 - 1);
```
해당 부분 변수도 이렇게 활용해주세요!

Reply:
리뷰 반영하였습니다 감사합니다!